docs/validators/setup/cli.md

```diff
@@ -64,30 +55,54 @@ Whenever you build or upgrade a node that includes a new hardfork, you must init
 
 ## Step 3. Start the node
 
-Run the following command to start a full (non-validator) Ronin node on the mainnet:
+To start a full (non-validator) Ronin node, run the following command:
+
+<Tabs groupId="networks">
+  <TabItem value="mainnet" label="Mainnet" default>
+
+  ```bash
+  ronin --http.api eth,net,web3,consortium --networkid 2020 --discovery.dns enrtree://AIGOFYDZH6BGVVALVJLRPHSOYJ434MPFVVQFXJDXHW5ZYORPTGKUI@nodes.roninchain.com --datadir /ronin/data \ --port 30303 --http --http.corsdomain '*' --http.addr 0.0.0.0 --http.port 8545 --http.vhosts '*' --ws --ws.addr 0.0.0.0 --ws.port 8546 --ws.origins '*' 
+  ```
+
+  </TabItem>
+  <TabItem value="testnet" label="Testnet" default>
+
+  ```bash
+  ronin --http.api eth,net,web3,consortium --networkid 2021 --discovery.dns enrtree://AJCNIAXQIPO55NW3QE2NUBBDMPYZDOQUCAEUS65NHQFMUUFES5KOW@saigon.nodes.roninchain.com --datadir /ronin/data \ --port 30303 --http --http.corsdomain '*' --http.addr 0.0.0.0 --http.port 8545 --http.vhosts '*' --ws --ws.addr 0.0.0.0 --ws.port 8546 --ws.origins '*' 
+  ```
+
+  </TabItem>
+</Tabs>
+
+This command starts `ronin` in full sync mode (default, can be changed with the `--syncmode` flag), causing it to download more data in exchange for avoiding processing the entire history of the Ronin network, which is very CPU-intensive.
```

Comment:
We currently does not support any mode than full. So please remove this line.

Reply:
@minh-bq I addressed your feedback, can you recheck?